signtol.m

```diff
@@ -1,4 +1,5 @@
 function s = signtol(x, tol)
-% Calculates the sign of x, given tolerance tol
+    % Calculates the sign of x, given tolerance tol
 
-s = (x>tol)-(x<-tol);
\ No newline at end of file
+    s = (x > tol) - (x < -tol);
```

Comment:
Make sure this does the same as before - Don't know if spaces count in this weird sorcery